src/app/Components/Navbar.tsx

```diff
@@ -0,0 +1,124 @@
+import MenuIcon from '@mui/icons-material/Menu';
+import {
+	AppBar,
+	Box,
+	Button,
+	Drawer,
+	Grid2,
+	IconButton,
+	List,
+	ListItem,
+	ListItemButton,
+	ListItemText,
+	Stack,
+	Toolbar,
+	Typography,
+} from '@mui/material';
+import Image from 'next/image';
+import React, { useState } from 'react';
+import LogoImage from '../../assets/images/mainLogo.png';
+
```

Comment:
Make navItems global
```suggestion
const navItems = ['Link 1', 'Link 2', 'Link 3'];
```

Reply:
This is temporary when we have the final items and links we can turn it into an object or an array. I think this should be fine for MVP 1?